low-level/message_handlers/disk_msg_handler.py

```diff
@@ -36,8 +36,6 @@
 from json_msgs.messages.sensors.expander_reset import ExpanderResetMsg
 from json_msgs.messages.sensors.hpi_data import HPIDataMsg
 from json_msgs.messages.sensors.node_hw_data import NodeIPMIDataMsg
-# Modules that receive messages from this module
-from message_handlers.logging_msg_handler import LoggingMsgHandler

```

Comment:
disk_msg_handler is invoked even anywhere today. I think its already disabled already, in which case lets not change code here

Reply:
It is used in DiskMonitor and NodeControllerMsgHandler currently, so I have removed only LoggingMsgHandler related code from this file for current ticket.
There is other deprecated code in this file, which can be removed in future ticket. like request/response handling regarding hpi, drive maanger, HDS etc.

Reply:
right, as discussed earlies, yes